front/app/modules/commercial/analytics/admin/utils/query.ts

```diff
@@ -1,25 +1,23 @@
 import moment, { Moment } from 'moment';
 import { IResolution } from 'components/admin/ResolutionControl';
 
-type ProjectFilter = { project: { id: string } };
+type ProjectFilter = { [key: string]: string };
 type EmptyObject = Record<string, unknown>;
```

Comment:
ProjectFilter needs fixing - it returns the following from the API: Filters field dimension_projects does not exist.